librz/demangler/demangler.c

```diff
@@ -0,0 +1,204 @@
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-FileCopyrightText: 2021 deroad <wargio@libero.it>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_demangler.h>
+#include <rz_util.h>
+#include <rz_list.h>
+#include <rz_libdemangle.h>
+
+static RzList *demangler_plugins = NULL;
```

Comment:
Just to wrap up what was discussed on mattermost:
1) no global
2) use RzDemangler object to keep track of demangler_plugins (and whatever can come later)
3) use `RZ_DEMANGLER_STATIC_PLUGINS` in config.h.in and initialize those plugins in `rz_demangler_new`